### PR TITLE
Ability to relate one allocation to many sites + dropping Input tables

### PR DESCRIPTION
**Pull request by thetrangdo**
This PR includes:

- DB scripts to add allocation-site bridge table + update allocation table to remove single key to sites table
- DB Entity updates
- API model updates
- DB scripts to drop [Input] tables

### CODE REVIEW

File: source/WesternStatesWater.WaDE.Accessors/Mapping/ApiProfile.cs [hunk 1]
```diff
@@ -22,12 +22,6 @@ public ApiProfile()
                 .ForMember(a => a.WaterSourceUUID, b => b.MapFrom(c => c.WaterSource.WaterSourceUuid))
                 .ForMember(a => a.MethodUUID, b => b.MapFrom(c => c.Method.MethodUuid))
                 .ForMember(a => a.VariableSpecificTypeCV, b => b.MapFrom(c => c.VariableSpecific.VariableSpecificCv))
```
Comment (Rhipeus) [inline source/WesternStatesWater.WaDE.Accessors/Mapping/ApiProfile.cs]: I expected to see a new mapping in this file for an EF.Site --> DTO.Site

File: source/WesternStatesWater.WaDE.DbUp/Scripts/001/072 Input Tables Drop.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿DROP TABLE IF EXISTS
```
Comment (Rhipeus) [inline source/WesternStatesWater.WaDE.DbUp/Scripts/001/072 Input Tables Drop.sql]: TIL - you can do mass table drops

File: source/WesternStatesWater.WaDE.Contracts.Api/Site.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+
+namespace WesternStatesWater.WaDE.Contracts.Api
+{
+    public class Site
```
Comment (Rhipeus) [inline source/WesternStatesWater.WaDE.Contracts.Api/Site.cs]: I also expected to see a new mapping for Accessor.Site --> Manager.Site

File: source/WesternStatesWater.WaDE.Accessors/Mapping/ApiProfile.cs (hunk 1 above)
Comment (thetrangdo) [reply source/WesternStatesWater.WaDE.Accessors/Mapping/ApiProfile.cs]: Added the EF to DTO mapping in the latest commit

File: source/WesternStatesWater.WaDE.Contracts.Api/Site.cs (hunk 3 above)
Comment (thetrangdo) [reply source/WesternStatesWater.WaDE.Contracts.Api/Site.cs]: Added in the latest commit

File: source/WesternStatesWater.WaDE.DbUp/Scripts/001/070 Core.AllocationBridgeSitesTable Create.sql [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+ALTER TABLE [Core].[AllocationBridge_Sites_fact]  WITH CHECK ADD  CONSTRAINT [FK_AllocationBridge_Sites_fact_AllocationAmounts_fact] FOREIGN KEY([AllocationAmountID])
+REFERENCES [Core].[AllocationAmounts_fact] ([AllocationAmountID])
+
+ALTER TABLE [Core].[AllocationBridge_Sites_fact] CHECK CONSTRAINT [FK_AllocationBridge_Sites_fact_AllocationAmounts_fact]
+
+ALTER TABLE [Core].[AllocationBridge_Sites_fact]  WITH CHECK ADD  CONSTRAINT [FK_AllocationBridge_Sites] FOREIGN KEY([SiteID])
+REFERENCES [Core].[Sites_Dim] ([SiteID])
+
+ALTER TABLE [Core].[AllocationBridge_Sites_fact] CHECK CONSTRAINT [FK_AllocationBridge_Sites]
+
```
Comment (Rhipeus) [inline source/WesternStatesWater.WaDE.DbUp/Scripts/001/070 Core.AllocationBridgeSitesTable Create.sql]: Please don't hate me but I forgot one thing that we should be doing after the `AllocationBridge_Sites_fact` creation and the `SiteId` column removal.  We need to actually migrate the existing data.  An `insert into AllocationBridge_Sites_fact` which selects from the `AllocationAmounts_fact` table should do it.  Let me know if you have any questions.

**Pull request merged**